### DISCUSSION TITLE
Should I Run a Ground Wire Between the High Voltage Box & the Low Voltage Box?

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#873] [Q&A]
I am building a DIY Nixie by converting an ET4.  To the extent possible, I am following the excellent guidance in this posting: https://github.com/tagyoureit/nodejs-poolController/discussions/801 as well as the wikis.

The transformers, circuit breakers, high voltage relays, chlorinator board, etc. will continue to be housed in the existing ET enclosure.  The Pi, MegaBas, relay hat, DIN rail terminal blocks, bridge rectifier, and eventually REM Chem circuitry will all be  housed in a separate fiberglass Altelix box.  The pool light transformer will remain in its own box, controlled through the Pi relay hat.  And the gas  heater will retain its current wiring, other than the low voltage relay hat control from the Pi.

Given this planned arrangement, is it be prudent to run a ground wire from the Altelix box (and DIN rails therein) to the ground bus at the breakers in the existing box?  I'm guessing  I should do this, but my expertise in this area is nearly non-existent.

Thanks for your help and guidance!

**Comment by rstrouse** [comment]
You will be running the ground from the panel to the psu nd the psu case is grounded to the rail.  The pi will have an isolated ground so no need for that but if you must you can run a ground between the rails but it is not required.

**Reply by FlaMike** [reply]
Thank you for your response!

If I understand correctly, because I do not plan to rewire the high voltage portion of the already installed ET, the transformers et al should remain grounded as you suggested.  And it does sound like I need to run a ground strap between the 2 boxes.  Of course, I will run new control wires to the high voltage relays, so I do have to access the high voltage section.

The DIN rails are being used solely to mount the Pi, Hats, terminal blocks, & bridge rectifier (already in a sealed package other than its connectors).  Since they won't be involved with the components electrically, there probably isn't a need to run a ground strap between then (though it seems harmless to do so).

Thanks again for your help!  Still drinking from a fire hose, but making progress :-)

**Reply by rstrouse** [reply]
Do not confuse grounding with bonding.  You do not need to bond the two panels.  The psu will take 120v so that will be grounded to the ground bar from the ET panel.  Other than that nothing more is required on the rail.

**Reply by FlaMike** [reply]
My bad.  I left out an important word in my response yesterday.  I meant to say "And it does NOT sound like I need to run a ground strap between the 2 boxes."

Nevertheless, thanks for keeping me on the straight & narrow!
